python/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ find_package(Python 3.9 REQUIRED COMPONENTS Interpreter Development.Module)
 include(CheckPythonModule)
 
 # Verify required Python packages are installed
+check_python_module("packaging")
```

Comment:
Drive-by fix to prevent a confusing failure if the `packaging` module is not installed. Then CMake would incorrectly conclude the version was not sufficient.

Reply:
Yeah my guess is I have it installed locally and it's included in the `manylinux` and Mac `cibuildwheel` environments by default.